test/fixtures/e2e/amp-viewer-integration/viewer.html

```diff
@@ -0,0 +1,388 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Viewer</title>
+    <meta
+      name="viewport"
+      content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no,minimal-ui"
+    />
+    <script src="/dist/v0/examples/amp-viewer-host.max.js"></script>
+    <style>
+      html,
+      body {
+        overflow: hidden;
+      }
+      body,
+      viewer {
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        overflow: hidden;
+        margin: 0;
+        padding: 0;
+      }
+
+      viewer {
+        background: #eee;
+      }
+
+      header {
+        position: absolute;
+        z-index: 1;
+        top: 0;
+        left: 0;
+        right: 0;
+        height: 50px;
+        background: #4285f4;
+        opacity: 0.7;
+        color: #fff;
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-start;
+        align-items: center;
+        padding: 0 8px;
+      }
+      header h1 {
+        font-size: 22px;
+      }
+
+      container {
+        position: absolute;
+        top: 50px;
+        left: 0;
+        right: 0;
+        bottom: 0;
+      }
+
+      container iframe {
+        border: 0;
+        margin: 0;
+        padding: 0;
+        width: 100%;
+        height: 100%;
+      }
+
+      .wait {
+        position: absolute;
+        z-index: 3;
+        top: 100px;
+        left: 20px;
+        font-size: 12px;
+      }
+    </style>
+    <script>
```

Comment:
Curious -- how much of this can be externalized into a shared JS file? It'd be nice to avoid divergence over time.